src/lib/yang/schema.lua

```diff
@@ -540,6 +540,40 @@ declare_initializer(init_typedef, 'typedef')
 declare_initializer(init_uses, 'uses')
 declare_initializer(init_value, 'value')
 
+
+-- 7.8.2.  The list's key Statement
```

Comment:
Is the real point here to make sure that key leaves are mandatory?  I feel like this code's goals could be accomplished in less.  I also think it needs to run after the "resolve" phase so that any groupings are pulled in as appropriate.  Would you be OK if I PR'd to this one to shorten this up?